nixos/modules/system/boot/systemd.nix

```diff
@@ -367,6 +367,16 @@ in
       '';
     };
 
+    kbrequestUnit = mkOption {
+      default = null;
+      type = types.nullOr types.str;
+      example = "rescue.target";
+      description = ''
+        Target that should be started when Alt+ArrowUp is pressed on the console;
+        see {manpage}`systemd.special(7)`.
+      '';
+    };
+
```

Comment:
possibly remove kbrequest.target in its entirety? see https://github.com/NixOS/nixpkgs/pull/419015#pullrequestreview-3111792107

```suggestion
```